packages/synapse-client/src/generatedSpecTest.test.ts

```diff
@@ -0,0 +1,70 @@
+import { describe, expect, expectTypeOf, test } from 'vitest'
+import {
+  EntityFromJSONTyped,
+  FileEntity,
+  instanceOfFileEntity,
+} from './generated'
+
+describe('Tests for the generated specification', () => {
+  test('instanceOfXxx works for a model with a concreteType', () => {
```

Comment:
Test the template change that checks that required enum properties have a valid value in `instanceOf...`